src/ripple/rpc/handlers/AccountInfo.cpp

```diff
@@ -39,6 +39,7 @@ namespace ripple {
 //           if true, only allow public keys and addresses. false, default.
 //   ledger_hash : <ledger>
 //   ledger_index : <ledger_index>
+//   signer_lists : <true | false>  // optional -- if true return SignerList(s)
```

Comment:
@ximinez: I tried your suggested phrasing.  I agree that it matched the phrasing on line 39, but I don't think it's as understandable to the reader.

Since consistency is a good thing, I could potentially change the description on line 39 to align with mine.  Perhaps the comment block should say:
```
//   account: <ident>,
//   strict: <bool> // optional -- if true only allow public keys and addresses.
//   ledger_hash : <ledger>
//   ledger_index : <ledger_index>
//   signer_lists : <bool> // optional -- if true return SignerList(s).
```
Thoughts?